ldclient/client.py

```diff
@@ -27,9 +27,19 @@
 
 
 class LDClient(object):
-    def __init__(self, config=None, start_wait=5):
+    def __init__(self, sdk_key=None, config=None, start_wait=5):
         check_uwsgi()
-        self._config = config or Config.default()
+
+        if config is not None and sdk_key is not None:
```

Comment:
I think this should check if the skd_key is set in both places, not if any config is passed in. Some existing clients might be using a modified config (changed timeouts, etc), but passing in the `sdk_key` separately.